_docs/_changelog.md

```
@@ -142,4 +142,19 @@ npm install express-rate-limit rate-limit-redis ioredis @types/ioredis --legacy-
     - [X] Create summaryGenerator.ts with generateConversationSummary and updateConversationSummary functions
     - [X] Wired up the summaryGenerator to conversations (sendMessage mutations - logic is going to be to generate summaries after 10+ messages and keep it updated on the go - 5 messages after that)
     - [X] Background processing, so we don't block the main thread
-    - [X] Test summaryGenerator on a file
\ No newline at end of file
+    - [X] Test summaryGenerator on a file
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)